File: src/main/java/com/currencies/mainpackage/api/dto/TicketResponse.java
package com.currencies.mainpackage.api.dto;

import lombok.Data;
import lombok.experimental.Accessors;

import java.sql.Timestamp;
import java.time.LocalTime;

@Data
@Accessors(chain = true)
public class TicketResponse {
    private String id;
    private Integer price;
    private Timestamp startFlightDate;
    private Timestamp endFlightDate;
    private LocalTime inFlight;
    private String fromPlace;
    private String toPlace;
}

File: src/main/java/com/currencies/mainpackage/api/dto/response/TicketResponse.kt
package com.currencies.mainpackage.api.dto.response

import com.currencies.mainpackage.api.dto.response.`object`.CityObject
import java.io.Serializable
import java.sql.Timestamp
import java.time.Duration

data class TicketResponse (
    val id: Long,

    val price: Int,

    val startFlightDate: Timestamp,

    val endFlightDate: Timestamp,

    val inFlight: Duration,

    val fromPlace: String,

    val toPlace: String

) : Serializable